test/cmd/BUILD

```diff
@@ -48,6 +48,7 @@ sh_library(
         "version.sh",
```

Comment:
this should maybe be a glob, too.

also the comment here doesn't really make sense.